src/main/java/core/basesyntax/service/impl/operation/SupplyOperation.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl.operation;
```

Comment:
The class `SupplyOperation` should not start with an empty line. Please remove the empty line at the beginning of the class definition.